### ISSUE TITLE
Ошибка линковки

### ISSUE THREAD

**Issue by SiliconD** [Flynston/yaregedit#1]
Я пытался запустить Ваш проект на своем компьютере, но столкнулся с ошибкой линковки. Может быть, Вы сможете подсказать мне, в чем может быть проблема? 
![image](https://cloud.githubusercontent.com/assets/16755057/25094835/9553b64e-23a1-11e7-8dc1-6cdef39b231b.png)

**Comment by Flynston** [comment]
Да, конечно, дело в том, что я хоть и использовал Qt фреймворк, но программировал не в Qt Creator'е, поскольку он автоматически генерит много лишнего (мое личное мнение). Я использовал 
утилиты qmake и make, а .pro файлы писал руками, поэтому перейдите из консоли в корень проекта и запустите команду qmake, она сгенерит вам Makefile, после этого запустите make, начнется сборка. Как только сборка закончится, скомпиленный файл будет лежать в папке build/release

**Comment by SiliconD** [comment]
Извиняюсь, что снова вас беспокою, но у меня не получается собрать решение. Я понимаю, что это, видимо, из-за разной разрядности системы и приложения, но не знаю, как это исправить. Я пробовал запускать x64 Native Tools Command Prompt, но там та же ошибка
![image](https://cloud.githubusercontent.com/assets/16755057/25129499/05ddccec-2447-11e7-95dc-985d55070312.png)

**Comment by SiliconD** [comment]
Сейчас я полностью пересобрал решение в x64 Command Prompt и получил следующие ошибки
![image](https://cloud.githubusercontent.com/assets/16755057/25129986/1c84942e-2449-11e7-9407-f8eddb87a546.png)
